packages/ffe-cards/less/theme.less

```diff
@@ -1,8 +1,15 @@
 :root {
     --ffe-v-cards-common-card-background-color: var(--ffe-farge-hvit);
     --ffe-v-cards-common-card-box-shadow-color: var(--ffe-farge-graa);
+    --ffe-v-cards-common-card-border-radius: 16px;
     --ffe-v-cards-text-color: var(--ffe-farge-moerkgraa);
     --ffe-v-cards-image-card-border-hover-color: var(--ffe-farge-vann);
+    --ffe-v-cards-stippled-border-color: var(--ffe-farge-vann);
+    --ffe-v-cards-stippled-icon-color: var(--ffe-farge-fjell);
+    --ffe-v-cards-stippled-title-color: var(--ffe-farge-fjell);
```

Comment:
De som går på textstilarna kan hete `--ffe-v-cards-common-card` brukes for alle kort i `.common-card-styling();`?

Reply:
Ja, de som er like på tvers av kortene kan godt samles sånn :)